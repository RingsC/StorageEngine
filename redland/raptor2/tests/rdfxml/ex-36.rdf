<?xml version="1.0" encoding="utf-8"?>
<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
         xmlns:ex="http://example.org/">
   <!-- Check parseType daml:Collection -->
   <rdf:Description rdf:about="http://example.org/resource">
      <ex:property rdf:parseType="daml:collection">
        <rdf:Description rdf:about="http://example.org/resource1"/>
      </ex:property>
   </rdf:Description>

</rdf:RDF>
