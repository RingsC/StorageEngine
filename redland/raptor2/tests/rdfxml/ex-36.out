_:genid1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.daml.org/2001/03/daml+oil#List> .
_:genid1 <http://www.daml.org/2001/03/daml+oil#first> <http://example.org/resource1> .
_:genid1 <http://www.daml.org/2001/03/daml+oil#rest> <http://www.daml.org/2001/03/daml+oil#nil> .
<http://example.org/resource> <http://example.org/property> _:genid1 .
